File: 3.py
def reverse_number(num):
    reversed_num = 0
    n = num
    while n > 0:
        reversed_num = reversed_num * 10 + n % 10
        n = n // 10
    return reversed_num

n = int(input("Введите количество чисел: "))

for _ in range(n):
    number = int(input("Введите число: "))
    reversed_num = reverse_number(number)
    difference = abs(number - reversed_num)
    print(f"|{number} - {reversed_num}| = {difference}")

File: 3.cs
using System;

class Program
{
    // Функция для реверса цифр числа
    static int ReverseNumber(int num)
    {
        int reversed = 0;
        while (num > 0)
        {
            reversed = reversed * 10 + num % 10;
            num /= 10;
        }
        return reversed;
    }

    static void Main()
    {
        Console.Write("Введите количество чисел: ");
        int n = int.Parse(Console.ReadLine());
        
        Console.Write("Введите числа через пробел: ");
        string[] input = Console.ReadLine().Split();
        
        Console.WriteLine("Результаты:");
        for (int i = 0; i < n && i < input.Length; i++)
        {
            int number = int.Parse(input[i]);
            int reversed = ReverseNumber(number);
            int difference = Math.Abs(number - reversed);
            Console.WriteLine($"|{number} - {reversed}| = {difference}");
        }
    }
}
